### ISSUE TITLE
Should we automate releases with Semantic Release?

### ISSUE THREAD

**Issue by mikeal** [request/request#2784]
https://github.com/semantic-release/semantic-release

I'm doing this in several projects now and it is quite nice.

This is the [smallest project](https://github.com/mikeal/webtorrent-component) doing it. I have a bunch of scripts to help walk people through the process of writing commit messages in a way that works for the automated releases.

**Comment by tylerdmace** [comment]
What are the potential downsides? This seems like something that is almost entirely upside.

**Comment by boneskull** [comment]
there's some overhead in getting contributors to use the commit message format, but [commitzen](https://npm.im/commitzen) helps.

there are probably other drawbacks, but the benefits are great.

**Comment by mikeal** [comment]
@boneskull yup, I usually add some automation to validate commit message and `npm run commit` to automate the commitizen commits https://github.com/mikeal/webtorrent-component/blob/master/package.json#L7
